src/VisualStudio/Core/SolutionExplorerShim/AnalyzersCommandHandler.cs

```diff
@@ -303,6 +311,23 @@ private void SetSeverityHandler(object sender, EventArgs args)
             }
         }
 
+        private void OpenDiagnosticHelpLinkHandler(object sender, EventArgs e)
+        {
+            if (_tracker.SelectedDiagnosticItems.Length != 1 ||
+                string.IsNullOrWhiteSpace(_tracker.SelectedDiagnosticItems[0].Descriptor.HelpLinkUri))
+            {
+                return;
+            }
+
+            string link = _tracker.SelectedDiagnosticItems[0].Descriptor.HelpLinkUri;
```

Comment:
will this do bing search if helplink doesn't actually exist using error ID? it would be nice if it does.